ui/src/app/pages/workspace/workspace-library.spec.tsx

```diff
@@ -30,16 +30,10 @@ describe('WorkspaceLibrary', () => {
   let publishedWorkspaceStubs = [];
   let PHENOTYPE_LIBRARY_WORKSPACES;
   let TUTORIAL_WORKSPACE;
-  let PUBLISHED_WORKSPACE;
 
-  const suffixes = [
-    ' Phenotype Library',
-    ' Tutorial Workspace',
-    ' Published Workspace',
```

Comment:
"Published Workspaces" is an abandoned feature from long ago.  We never fully launched it, so I'm removing its remnants now, to avoid confusion and complexity.